docs/src/concepts.md

```diff
@@ -0,0 +1,6 @@
++ Most pipelines track some information about the lab, including the facilities, experiment rigs, and users. All interviewed labs have some version of these elements. They also have custom interfaces and GUIs for data entry.
+
++ Over the past few years, several labs have developed DataJoint-based pipelines for lab management. Our team collaborated with several of them during their projects. Additionally, we interviewed these teams to understand their experiment workflow, associated tools, and interfaces. These teams include: + International Brain Laboratory + BrainCoGs (Princeton Neuroscience Institute), Python pipeline, MATLAB pipeline + MoC3 (Columbia Zuckerman Institute) + Churchland Lab + Costa Lab (private repository) + Hillman Lab
+Development and validation
+
++ Through our interviews and direct collaboration on the precursor projects, we identified the common motifs in the lab schemas to create the Lab Management Element. This element works for diverse downstream pipelines and is always used in combination with other elements for specific experiments. As such, it is validated jointly with the acquisition elements such as the Extracellular Array Electrophysiology Element and Calcium Imaging Element.
```

Comment:
This was framed more as 'development' than concepts, so we need to edit for the current purpose.

You also copy-pasted bullets that are presented here as text.

```suggestion
# Concepts

Most pipelines track some information about the lab, including the facilities, experiment rigs, and users. This can help track which users lab members are associated with which projects or who is responsible for equipment.

Through our interviews and direct collaboration on the precursor projects, we identified the common motifs in the lab schemas. This Element works for diverse downstream pipelines and is always used in combination with other Elements for specific experiments. As such, it is validated jointly with the acquisition Elements such as
[Extracellular Array Electrophysiology](https://datajoint.com/docs/elements/element-array-ephys) 
and 
[Calcium Imaging](https://datajoint.com/docs/elements/element-calcium-imaging).

## Development

Over the several years, many labs have developed DataJoint-based pipelines for lab management. The DataJoint team collaborated with several and interviewed these teams to understand their experiment workflow, associated tools, and interfaces. These teams include: 

- International Brain Laboratory
- BrainCoGs (Princeton Neuroscience Institute), Python pipeline, MATLAB pipeline 
- MoC3 (Columbia Zuckerman Institute) 
- Churchland Lab 
- Costa Lab (private repository) 
- Hillman Lab
```